docker/files/amplify/amplify-meta.json.base

```diff
@@ -41,12 +38,12 @@
       "providerPlugin": "awscloudformation",
       "service": "Lambda"
     },
-    "getUserReputation": {
```

Comment:
Same question as above, we don't need `getUserReputation` anymore ?

Also, I see it listed in `team-provider-info`, which makes me suspect this was done in error, so you need to go back and fix all places where this occurs

Reply:
I presume that `getUserReputation` is required. It was partially removed during a rebase.